### PR TITLE
refactor & optimize property.json

### PR DESCRIPTION
**Pull request by plutoless**

### CODE REVIEW

File: server/internal/worker.go [hunk 1]
```diff
@@ -183,9 +186,9 @@ func (w *Worker) start(req *StartReq) (err error) {
 }
 
 func (w *Worker) stop(requestId string, channelName string) (err error) {
-	slog.Info("Worker stop start", "channelName", channelName, "requestId", requestId, logTag)
+	slog.Info("Worker stop start", "channelName", channelName, "requestId", requestId, "pid", w.Pid, logTag)
 
-	err = syscall.Kill(w.Pid, syscall.SIGTERM)
+	err = syscall.Kill(-w.Pid, syscall.SIGTERM)
```
Comment (wangyoucao577) [inline server/internal/worker.go:191]: why negative?

File: server/main.go [hunk 2]
```diff
@@ -54,15 +51,6 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Set graph name map
-	internal.SetGraphNameMap()
```
Comment (wangyoucao577) [inline server/main.go:58]: Same part in `.env` can be removed either?

File: server/internal/worker.go (hunk 1 above)
Comment (plutoless) [reply server/internal/worker.go:191]: it means to kill all processes under the process group.
as we are starting the command via "sh -c", it's in fact two processes here, which leaves chances that one is killed while the other is left orphan. so we need to group these two processes and kill them as a group.

File: server/internal/http_server.go [hunk 3]
```diff
@@ -358,6 +358,18 @@ func (s *HttpServer) output(c *gin.Context, code *Code, data any, httpStatus ...
...
+	result := ""
+	for i, elem := range elements {
+		if i > 0 {
+			result += sep
+		}
+		result += elem
+	}
+	return result
+}
+
```
Comment (sunshinexcode) [inline server/internal/http_server.go:372]: can use strings.Join instead

**Comment by sunshinexcode** [comment]
LGTM

**Review by sunshinexcode — APPROVED** [review]
LGTM

**Review by wangyoucao577 — APPROVED** [review]

**Review by sunshinexcode — APPROVED** [review]
LGTM

**Pull request merged**